### DISCUSSION TITLE
Use hypothesis to find bugs in the lilypond parser?

### DISCUSSION THREAD

**Discussion by jgarte** [Abjad/abjad#1504] [Ideas]
@trevorbaca @josiah-wolf-oberholtzer 

What do you think of using hypothesis to test and find bugs in the lilypond parser?

I think that it might help find bugs similar to this one:

https://github.com/Abjad/abjad/issues/1471

**Comment by josiah-wolf-oberholtzer** [comment]
@jgarte Recommend trying it out yourself and reporting back if you find something concretely useful.
